docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_3.md

```diff
@@ -197,6 +197,28 @@ image.save("sd3_hello_world.png")
 
 Check out the full script [here](https://gist.github.com/sayakpaul/508d89d7aad4f454900813da5d42ca97).
 
+## Tiny AutoEncoder for Stable Diffusion 3
+
+Tiny AutoEncoder for Stable Diffusion (TAESD3) was introduced in [madebyollin/taesd](https://github.com/madebyollin/taesd) by Ollin Boer Bohan. It is a tiny distilled version of Stable Diffusion 3's VAE that can quickly decode the latents in a [`StableDiffusion3Pipeline`] almost instantly.
+
+To use with Stable Diffusion 3:
+
+```python
+import torch
+from diffusers import StableDiffusion3Pipeline, AutoencoderTiny
+
+pipe = StableDiffusion3Pipeline.from_pretrained(
+    "stabilityai/stable-diffusion-3-medium-diffusers", torch_dtype=torch.float16
+)
+pipe.vae = AutoencoderTiny.from_pretrained("madebyollin/taesd3", torch_dtype=torch.float16)
+pipe.vae.config.shift_factor = 0.0
```

Comment:
For this line, we might want to fix the underlying incompatibility (`AutoencoderTiny` doesn't have a `shift_factor` which SD3 pipeline is surprised by) rather than baking the fix into the docs :)

Reply:
That sounds good to me. 

How about merging this PR and then I can take care of the fix in another PR?

Reply:
Sounds good to me! Just need to update the docs in the fix PR and we're good